src/simplewallet/simplewallet.cpp

```diff
@@ -1462,6 +1464,72 @@ bool simple_wallet::handle_command_line(const boost::program_options::variables_
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::prompt_for_new_node()
+{
+    bool connect = false;
+    std::string node_url;
+
+    message_writer() << tr("Do you want to connect to a remote node?");
+    std::string confirm = command_line::input_line(tr("(Y/Yes/N/No): "));
+    if(std::cin.eof())
+    {
+      LOG_ERROR("Unexpected std::cin.eof() - Exited simple_wallet::prompt_for_new_node()");
+      return false;
```

Comment:
IIRC wallet functions must return true.

Reply:
I can see the ``simple_wallet::init()`` has a couple of `return false`'es as well, so should be fine?

Reply:
Sorry, I meant wallet functions mapped to a command.